doc-content/enterprise-only/fuse/ba-karaf-xml-uninstall-proc.adoc

```diff
@@ -1,17 +1,26 @@
-[id='features-xml-removing']
-= Removing features XML files
-If your installation contains an older version of the {PRODUCT} features XML files (for example, `kie-karaf-features-<VERSION>-features.xml`), you must remove these files and all associated files before installing the most recent features XML files.
+[id='ba-karaf-xml-uninstall-proc']
+= Uninstalling obsolete {PRODUCT} features XML files on Karaf
+
+If your installation contains older versions of the {PRODUCT} features XML files (for example, `kie-karaf-features-<VERSION>-features.xml`), you must remove these files and all associated files before installing the most recent features XML files.
 
 .Prerequisites
-* Obsolete features XML files exist in your installation.
-COMMENT: How do I know if I have obsolete files? How do I know what features use the obsolete files?
+* Obsolete features XML files exist in your installation Apache Karaf installation.
+
 
 .Procedure
-. Enter the following command to start the {FUSE_LONG} console where COMMENT: What is INSTALLATION_DIRECTORY? The Fuse home? The Karaf home?
+. Enter the following commands to determine whether your installation contains obsolete {PRODUCT} features XML files:
++
+[source]
+----
+$ JBossFuse:karaf@root> feature:repo-list
+$ JBossFuse:karaf@root> feature:list
+----
+
+. Enter the following command to start the {FUSE_LONG} console where `FUSE_HOME` is the Fuse installation directory:
```

Comment:
If you use the "where X is Y" format, it needs to come after "following command, " and with the comma ("the following command, where X is Y:"). If other text comes between it, it makes it a bit of a dangling modifier. If rewording to achieve that is too awkward, you could go with the common IBM approach of "where ..." lower case after the snippet. I don't care for that much, so the IBM alternative that you can do either inline above or after the snippet is just "Replace X with the Fuse installation ...". That would go fine in the step itself (which I prefer for simple replacements) or after the snippet.

This comes up several times, so just noting here.